compiler-rt/lib/scudo/standalone/tests/combined_test.cpp

```diff
@@ -333,6 +333,12 @@ void ScudoCombinedTest<Config>::BasicTest(scudo::uptr SizeLog) {
 
   Allocator->printStats();
   Allocator->printFragmentationInfo();
+
+  {
+    char buffer[256] = {0};
+    EXPECT_LE(0, Allocator->getStats(buffer, sizeof(buffer)));
+    EXPECT_LE(0, Allocator->getFragmentationInfo(buffer, sizeof(buffer)));
```

Comment:
Can we change these to  EXPECT_GT(Allocator->..., 0);

Same as the uses in the tests

Reply:
Done.